alerta/utils/api.py

```diff
@@ -108,15 +107,18 @@ def process_action(alert: Alert, action: str, text: str) -> Tuple[Alert, str, st
                 logging.error("Error while running action plugin '{}': {}".format(plugin.name, str(e)))
         if updated:
             try:
-                alert, action, text = updated
+                try:
+                    alert, action, text = updated    #if timeout does not get returned, try default behavior
+                except Exception:
```

Comment:
Ideally, we should catch TypeError here to catch unpacking errors and not a generic exception.

Reply:
also might be cleaner, since `updated` is a tuple, to check the length and run logic based on that instead of doing try...except:
```
if len(updated) == 3:
  alert, action, text = updated   
elif len(updated) == 4: 
  alert, action, text, timeout = updated
else:
  alert = updated
```

Reply:
This is a much cleaner solution.  Thank you!